emission/net/ext_service/habitica/proxy.py

```diff
@@ -26,8 +27,11 @@ def habiticaRegister(username, email, password, our_uuid):
   #want to access their Habitica account from the browser
   #Need to create a way from them to retrieve username/password
   habitica_user_table = edb.get_habitica_db()
-  habitica_user_table.insert({'user_id': our_uuid, 'habitica_username': username, 
-    'habitica_password': password, 'habitica_id': user_dict['data']['_id'], 
+  habitica_user_table.insert({'user_id': our_uuid, 
+    'last_timestamp': arrow.utcnow().timestamp,
```

Comment:
There's a standard technique that I use for this with the other pipeline states.
Look at `emission/core/wrapper/pipelinestate.py`, `emission/storage/pipeline_queries.py` for examples of where it is defined and `emission/analysis/intake/segmentation/trip_segmentation.py` - e.g. `time_query = epq.get_time_range_for_segmentation(user_id)` for where it is used.